addons/ai/XEH_unitInit.sqf

```diff
@@ -1,45 +1,35 @@
-#include "script_component.hpp"
-
-params [["_unit", objNull]];
-
-if (isNull _unit || {isPlayer _unit}) exitWith {};
-
-_unit addEventHandler ["Local", FUNC(handleLocality)];
-
-if (!local _unit) exitWith {};
-
-[{
-    [{
-        params ["_unit"];
-
-        [_unit] call FUNC(assignLoadout);
-
-        _unit disableAI "SUPPRESSION";
```

Comment:
I agree with removing this, although I do not know why it was removed exactly in the first place.